examples/api500/storage_system.rb

```diff
@@ -73,10 +76,13 @@
 
 storage_system = storage_system_class.new(@client, hostname: storage_system['hostname'])
 storage_system.retrieve!
-
 port = storage_system['ports'].find { |item| item['protocolType'].downcase.include?('fc') } # find first correct protocolType for using our fc network
 if port
-  fc_network = OneviewSDK::API500::C7000::FCNetwork.get_all(@client).first
+  fc_network = if @client.api_version == 600
+                 OneviewSDK::API600::C7000::FCNetwork.get_all(@client).first
+               else
+                 OneviewSDK::API500::C7000::FCNetwork.get_all(@client).first
+               end
   puts "\n Adding a fc network named '#{fc_network['name']}' with uri='#{fc_network['uri']}' to the storage system."
   port['expectedNetworkUri'] = fc_network['uri']
   port['expectedNetworkName'] = fc_network['name']
```

Comment:
Is it ok if this file is under api500 folder? We need to create another common folder or have a similar file under api600 so that there is no confusion/ambiguity.

Reply:
yes already discussed with Sijeesh regarding this. And left out comments. As we don't have any change in storage systems, we are gong to keep the same file to reduce redundancy

Reply:
We can keep the same file but it should not be inside api500 folder is my point of view.

Reply:
we should avoid duplicate code wherever possible in my opinion and if the code is same, then i don't see any reason to create another folder and increase the sdk project size.